src/Microsoft.VisualStudio.Services.Agent/Blob/DedupManifestArtifactClientFactory.cs

```diff
@@ -222,6 +237,8 @@ private DedupManifestArtifactClientFactory()
                 cancellationToken: cancellationToken,
                 continueOnCapturedContext: false);
 
+            dedupStoreHttpClient.SetRedirectTimeout(GetRedirectTimeoutFromClientSettings(clientSettings));
```

Comment:
what happens if this value is null? Does it fallback to using env variable?

Reply:
env always comes first. if there's no env, null will reset it to the default

Reply:
https://dev.azure.com/mseng/AzureDevOps/_git/AzureDevOps?path=/BlobStore/Client/WebApi/DedupStoreHttpClient.cs&version=GBmaster&line=741&lineEnd=742&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

Reply:
Got it. Yeah, we will have to upgrade the vss-api-netcore pkg as well, in the next PR.